autoware_launch/launch/components/tier4_control_component.launch.xml

```diff
@@ -42,5 +43,7 @@
     <arg name="external_cmd_selector_param_path" value="$(find-pkg-share autoware_launch)/config/control/external_cmd_selector/external_cmd_selector.param.yaml"/>
     <arg name="aeb_param_path" value="$(find-pkg-share autoware_launch)/config/control/autonomous_emergency_braking/autonomous_emergency_braking.param.yaml"/>
     <arg name="enable_autonomous_emergency_braking" value="$(var enable_autonomous_emergency_braking)"/>
+    <arg name="enable_predicted_path_checker" value="$(var enable_predicted_path_checker)"/>
+    <arg name="predicted_path_checker_param_path" value="$(find-pkg-share autoware_launch)/config/control/predicted_path_checker/predicted_path_checker.param.yaml"/>
```

Comment:
You are sending this `enable_predicted_path_checker` argument to the `control.launch.py` but you've forgotten to add it as a launch argument in the [previous PR](https://github.com/autowarefoundation/autoware.universe/pull/2528/files#r1342120806):

https://github.com/brkay54/autoware.universe/blob/1f2f0aaf82f103bef748e54448d2b2bf1fb93eef/launch/tier4_control_launch/launch/control.launch.py#L407

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5186 is merged.